reTurn/client/TurnAsyncSocket.hxx

```diff
@@ -144,6 +148,8 @@ private:
    bool mCloseAfterDestroyAllocationFinishes;
    resip::Mutex mMutex;
 
+   std::string mSoftware;
```

Comment:
Let's use resip::Data as the storage type, to be consistent with other strings in this class (ie. mUsername, etc.)